messenger/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .settings import *
```

Comment:
Sure, I'll take a look at the PR and try to find the best way to integrate your work. This was actually a concern of mine since testing on windows was very minimal, and I had only run them on a VM on OSX. If you could lend assistance with running the new code and providing feedback that would be much appreciated!